packages/rancher-monitoring/generated-changes/patch/templates/exporters/kubelet/servicemonitor.yaml.patch

```diff
@@ -1,6 +1,12 @@
 --- charts-original/templates/exporters/kubelet/servicemonitor.yaml
 +++ charts/templates/exporters/kubelet/servicemonitor.yaml
-@@ -3,7 +3,7 @@
+@@ -1,9 +1,12 @@
+-{{- if .Values.kubelet.enabled }}
++{{- if (and (not .Values.kubelet.enabled) .Values.hardenedKubelet.enabled) }}
++{{ required "Cannot set .Values.hardenedKubelet.enabled=true when .Values.kubelet.enabled=false" "" }}
++{{- end }}
++{{- if (and .Values.kubelet.enabled (not .Values.hardenedKubelet.enabled)) }}
```

Comment:
hardenedKubelet is treated as an optional add-on to kubelet; you cannot enable hardenedKubelet unless kubelet is enabled.

If hardenedKubelet is enabled, it disables the default ServiceMonitor that is created for kubelet monitoring in favor of one that routes through the proxy.